### PR TITLE
Adds Geospatial component to the query API to limit results

### PR DESCRIPTION
**Pull request by siddddhant**
Query API takes the following parameters
lat - Latitude - Required
lng - Longitude - Required
radius - Distance in meters. All stores with the radius are returned in result - Optional Default value is 1100 meters

Sample Request: `[BASE_URL]/v2/query?lat=13.0827&lng=80.2707&radius=5000`

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -119,4 +119,13 @@ app.get("/v1/query", async (req, res) => {
   res.send(await stores.findAllStores());
 });
 
+app.get("/v2/query", async (req, res) => {
+  let query = {
+    lat: req.query.lat,
+    lng: req.query.lng,
+    radius: req.query.radius,
+    page: req.query.page}
+  res.send(await stores.findNearbyStores(query));
```
Comment (arjun27) [inline server/index.js:128]: can we use the new format as in #170 
```
{ results: stores.findNearbyStores(), location: { // default lat-lng } }
```

File: server/index.js [hunk 2]
```diff
@@ -119,4 +119,13 @@ app.get("/v1/query", async (req, res) => {
   res.send(await stores.findAllStores());
 });
 
+app.get("/v2/query", async (req, res) => {
+  let query = {
+    lat: req.query.lat,
+    lng: req.query.lng,
+    radius: req.query.radius,
+    page: req.query.page}
```
Comment (arjun27) [inline server/index.js:127]: page doesn't seem to be used?

Comment (siddddhant) [reply server/index.js:127]: It is for future use cases. We can order the result by the distance and use pagination inbuilt into backend

Comment (arjun27) [reply server/index.js:127]: sounds good

**Pull request merged**